EMT_data_analysis/analysis_scripts/Analysis_tools.py

```diff
@@ -145,6 +144,17 @@ def plot_area_at_glass_all_data(df, figs_dir, out_type):
     plt.legend(bbox_to_anchor=(1.05, 1.0), loc='upper left') 
     plt.savefig(rf'{figs_dir}/Area_at_the_glass_over_time_MIP_n{n_a}.{out_type}', transparent=True, dpi=600)
 
+    plot_tools.plot_examples(
```

Comment:
Since an individual example got added earlier in the code , the making of this directory needs to be moved up to avoid an error
```suggestion
    Path(rf'{figs_dir}/Individual_Examples').mkdir(exist_ok=True, parents=True)
    plot_tools.plot_examples(
```

Reply:
This used to be done down around line 353 so once this line is added here is can be removed from that later line

Reply:
I have added the code to the area at glass function, but I believe it should be kept in both functions in case we only want to run one function but not the other.